src/lib/Wheel.svelte

```diff
@@ -24,12 +23,17 @@
   const spinWheel = () => {
     spinDeg += Math.random() * 8 + 3;
   };
-
+  let segmentColors = [...colors]
+  
+  const colorArrayLengthMatcher =()=>{
+    if(colors.length !== items.length){
+      segmentColors = Array.from({length: items.length}, (_,i)=>colors[i % colors.length])
```

Comment:
does this work? shouldn't it be:

```typescript
segmentColors.concat(Array.from({length: items.length}, (_,i)=>colors[i % colors.length]));
```

or something to that effect so it only adds more colors to the existing segmentColors array? It looks like currently it will just reassign to an array equal to the difference between the two arrays

Reply:
thanks for going over it with me. Looks Good 👍